trajopt_optimizers/trajopt_optimizers_test_suite/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>trajopt_optimizers_test_suite</name>
+  <version>0.1.0</version>
+  <description>The trajopt_optimizers_test_suite package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="randall.kliman@swri.org">Randall Kliman</maintainer>
```

Comment:
Why don't you put yourself as the author and Levi as the maintainer. This email will be dead in a few months when your internship ends.